junit5-engine/src/main/java/org/junit/gen5/engine/junit5/discovery/DiscoverySelectorResolver.java

```diff
@@ -59,15 +59,10 @@ public void resolveSelectors(EngineDiscoveryRequest request) {
 			String packageName = selector.getPackageName();
 			findAllClassesInPackage(packageName, isScannableTestClass).stream().forEach(this::resolveTestClass);
 		});
-		request.getSelectorsByType(ClassSelector.class).forEach(selector -> {
-			resolveTestClass(selector.getTestClass());
-		});
-		request.getSelectorsByType(MethodSelector.class).forEach(selector -> {
-			resolveTestMethod(selector.getTestClass(), selector.getTestMethod());
-		});
-		request.getSelectorsByType(UniqueIdSelector.class).forEach(selector -> {
-			resolveUniqueId(selector.getUniqueId());
-		});
+		request.getSelectorsByType(ClassSelector.class).forEach(selector -> resolveTestClass(selector.getTestClass()));
+		request.getSelectorsByType(MethodSelector.class).forEach(
+			selector -> resolveTestMethod(selector.getTestClass(), selector.getTestMethod()));
+		request.getSelectorsByType(UniqueIdSelector.class).forEach(selector -> resolveUniqueId(selector.getUniqueId()));
```

Comment:
I did this on purpose for better readability (which is of course subjective). Can you please change it back?